test/gradle-test-wrapper/build.gradle

```diff
@@ -1,8 +1,10 @@
 plugins {
     id('java')
+    id("com.adarshr.test-logger") version "4.0.0"
```

Comment:
It makes it easier/possible to see the progress of test runs. For example:

https://github.com/casid/jte/actions/runs/6792475804/job/18465818661#step:7:137

Reply:
I would rather remove this, as it is an extra dependency that needs to be kept up to date, while it only provides minimal extra convenience.

Reply:
Done in 5170ff2.